adafruit_ds3231.py

```diff
@@ -17,21 +17,26 @@
 
 **Hardware:**
 
-* Adafruit `DS3231 Precision RTC FeatherWing <https://www.adafruit.com/products/3028>`_
-  (Product ID: 3028)
+* `Adafruit DS3231 Precision RTC FeatherWing
+  <https://www.adafruit.com/products/3028>`_ (Product ID: 3028)
+
+* `Adafruit DS3231 RTC breakout
+  <https://www.adafruit.com/products/3013>`_ (Product ID: 3013)
+
+* `Adafruit ChronoDot - Ultra-precise Real Time Clock - v2.1
+  <https://www.adafruit.com/products/255>`_ (Product ID: 3013)
```

Comment:
This should be updated, yes?
```suggestion
  <https://www.adafruit.com/products/255>`_ (Product ID: 255)
```

Reply:
Thanks :)

Reply:
Changes in https://github.com/adafruit/Adafruit_CircuitPython_DS3231/pull/35/commits/03c7efa48c8e39533b2305e0fd68f64649cc1871